### PR TITLE
Move PillowError actions to django admin (and remove old view & app)

### PR DESCRIPTION
**Pull request by emord**
##### SUMMARY
We already have PillowError documents in django admin `/admin/pillow_retry/pillowerror`, but we have this old view that displays the errors as well.

I moved deleting and resetting attempts to admin actions. I made a call to ignore the ability to email the pillow error to support because I don't think that's been used in quite a while.

### CODE REVIEW

File: corehq/apps/reports/templates/reports/standard/base_template.html [hunk 1]
```diff
@@ -30,7 +30,6 @@
     <!-- Scripts that are specific to a single report or interface -->
     <script src="{% static 'data_interfaces/js/case_management.js' %}"></script>
     <script src="{% static 'data_interfaces/js/archive_forms.js' %}"></script>
-    <script src="{% static 'hqpillow_retry/js/pillow_errors.js' %}"></script>
```
Comment (emord) [inline corehq/apps/reports/templates/reports/standard/base_template.html]: A random staff private report that includes javscript on every report? classic reporting code

Comment (orangejenny) [reply corehq/apps/reports/templates/reports/standard/base_template.html]: The reports UI has rather a lot of baggage at this point.

**Review by orangejenny — APPROVED** [review]

File: settings.py [hunk 2]
```diff
@@ -508,7 +507,7 @@
 GET_URL_BASE = 'dimagi.utils.web.get_url_base'
 
 # celery
-CELERY_BROKER_URL = 'redis://localhost:6379/0'
+BROKER_URL = CELERY_BROKER_URL = 'redis://localhost:6379/0'
```
Comment (snopoke) [inline settings.py]: why is this setting getting added back?

**Comment by snopoke** [comment]
Thanks, this has been on my kill list for a while.

File: corehq/apps/hqpillow_retry/management/commands/run_pillow_retry_queue.py [hunk 3]
```diff
@@ -1,59 +0,0 @@
-from datetime import datetime
```
Comment (snopoke) [inline corehq/apps/hqpillow_retry/management/commands/run_pillow_retry_queue.py]: @emord this is the command to run the pillow retry queue - we need it. I'm going to restore it.

Comment (emord) [reply corehq/apps/hqpillow_retry/management/commands/run_pillow_retry_queue.py]: Oof. really sorry about that (https://github.com/dimagi/commcare-hq/pull/25854)

**Pull request merged**